backend/app/tests/utils/test_image_utils_with_storage.py

```
@@ -0,0 +1,137 @@
+
+    # 验证文件已上传到模拟存储
+    file_path = f"avatars/{user_id}.png"
+    assert file_path in mock_storage.files
+    assert len(mock_storage.files[file_path]) > 0
+
+
+@pytest.mark.asyncio
+@patch("app.utils.image_utils.get_storage_service")
+@patch("app.utils.image_utils.httpx.AsyncClient")
+async def test_get_github_avatar_with_storage(
+    mock_client: MagicMock, mock_get_storage: MagicMock
```

Ruff (0.11.9)

Unused function argument: `mock_client`

(ARG001)